README.md

```diff
@@ -38,7 +38,7 @@ For documentation, please visit [Container Storage Modules documentation](https:
 * [Troubleshooting](#troubleshooting)
 
 ## Supported CSI Drivers
-Currently, CSM Volume Group Snapshotter provides support to create and delete volume group snapshots for PowerFlex array. 
+Currently, CSM Volume Group Snapshotter provides support to create and delete volume group snapshots for PowerFlex and PowerStore arrays. 
```

Comment:
If you want to be more precise, I would rephrase the last part of the sentence as " delete volume group snapshots on Dell PowerFlex and Dell PowerStore storage platforms".

Reply:
updated